src/lib.rs

```diff
@@ -20,6 +20,7 @@ pub mod logger;
 pub mod messages;
 pub mod paths;
 pub mod profile;
+mod retry_manager;
 pub mod tui;
 pub mod ui;
 
```

Comment:
```suggestion
pub mod messages;
pub mod paths;
pub mod profile;
pub mod tui;
pub mod ui;

mod retry_manager;
```

Separate public and private modules.